CMakeLists.txt

```diff
@@ -192,6 +192,19 @@ set(c_util_h_files
     ./inc/c_util/uuid_string.h
 )
 
+if(WIN32)
+#add worker_thread only on Windows (until we fix it)
+set(c_util_c_files
+    ${c_util_c_files}
+    ./src/worker_thread.c
+)
+
+set(c_util_h_files
+    ${c_util_h_files}
+    ./inc/c_util/worker_thread.h
+)

```

Comment:
nitpick: can you indent between the if()...endif().  Keep the cmakelist pretty.

Reply:
It's very pretty now :-)